pos_frequency_pipeline.py

```diff
@@ -2,7 +2,68 @@
 Implementation of POSFrequencyPipeline for score ten only.
 """
 
+import json
+from pathlib import Path
+import re
+
+from pymystem3 import Mystem
+
+from constants import ASSETS_PATH
+from pipeline import CorpusManager
+from visualizer import visualize
+
 
 class POSFrequencyPipeline:
     def __init__(self, assets):
+        self.assets = assets
```

Comment:
the interface should be the same as the Pipeline - accepting corpus_manager as the only argument